docs/census-open-data-registry.yaml

```diff
@@ -0,0 +1,37 @@
+Name: CZ CELLxGENE Discover Census
+Description: The Census provides efficient computational tooling to access, query, and analyze all single-cell RNA data from CZ CELLxGENE Discover.
+Documentation: https://chanzuckerberg.github.io/cellxgene-census/
+Contact: cellxgene@chanzuckerberg.com
+ManagedBy: "[Chan Zuckerberg Initiative](http://www.chanzuckerberg.com/)"
```

Comment:
... Foundation

Reply:
I believe that is not used anywhere else in the Census repo (for instance, the copyright doesn't mention it). Maybe we should ask branding?

Reply:
It is definitely used in our copyright and legal names.  See, for example, our code copyright.  Also listed in our Legal Terms on the cellxgene web site.

I'm confident we need to say Chan Zuckerberg Initiative Foundation

Reply:
I was referring to the [census copyright](https://github.com/chanzuckerberg/cellxgene-census/blob/main/LICENSE). In that case, we probably want to update that as well for consistency.

Reply:
oh, good catch!  

Oddly, it looks like it is correct in the TileDB-SOMA code copyright, but not in our own repo :blush: 

I'll file a separate bug for this issue